test/puppet-mode-test.el

```diff
@@ -601,6 +601,82 @@ package { 'bar':
     (should (local-variable-p 'imenu-create-index-function))
     (should (equal imenu-create-index-function #'puppet-imenu-create-index))))
 
+
+;;;; Indentation
+
+(defmacro flycheck-ert-with-temp-buffer (&rest body)
```

Comment:
If you say so, I just thought to give credit where it's due (or rather since I'm lazy I just didn't want to touch a thing.) Guess a comment will suffice here.